test/setupTest.js

```diff
@@ -0,0 +1,4 @@
+import { configure } from 'enzyme';
```

Comment:
As my coding style, spaces around `{` and `}` should be removed

Reply:
Will do.